qtfred/src/mission/dialogs/CommandBriefingDialogModel.cpp

```diff
@@ -28,16 +28,16 @@ void CommandBriefingDialogModel::initializeData()
 		strcpy_s(_wipCommandBrief.background[1], "<default>");
 	}
 
-	if (_wipCommandBrief.num_stages <= 0) {
-		_currentStage = 0;
-	} else {
-		_currentStage = 1;
-	}
+	_currentStage = 0;
 
 	int i;
 
 	for (i = 0; i < _wipCommandBrief.num_stages; i++) {
 		_wipCommandBrief.stage[i] = Cur_cmd_brief->stage[i];
+		strcpy_s(_wipCommandBrief.stage[i].ani_filename, Cur_cmd_brief->stage[i].ani_filename);
+		_wipCommandBrief.stage[i].text = Cur_cmd_brief->stage[i].text;
+		_wipCommandBrief.stage[i].wave = Cur_cmd_brief->stage[i].wave;
+		strcpy_s(_wipCommandBrief.stage[i].ani_filename, Cur_cmd_brief->stage[i].wave_filename);
```

Comment:
Looks like a copy-paste error... should the first argument be `wave_filename`?

Reply:
Thanks for catching that